pallets/stateful-storage/src/lib.rs

```diff
@@ -235,14 +234,21 @@ pub mod pallet {
 
 			match updated_page.is_empty() {
 				true => {
-					ChildTreeStorage::kill(&state_owner_msa_id, storage_key);
+					StatefulChildTree::kill::<Page<T::MaxItemizedPageSizeBytes>>(
```

Comment:
kill doesn't need to have a type. We can remove the generic type from kill function

Reply:
@JoeCap08055  can include this in his branch